polyfills/console/error/config.json

```diff
@@ -4,6 +4,7 @@
 	],
 	"browsers": {
 		"ie": "6 - 11",
+		"ie_mob": "* - 9",
```

Comment:
For this kind of range we prefer `<10`.  This is because 10 is the significant number - the version in which the feature was first supported.  `0 - 9` requires you to remember a) that semver requires spaces around the `-`, which is a gotcha, and b) that 9 is the last version that does not have the feature, not the first version that does.